packages/apollo/src/index.ts

```diff
@@ -18,13 +23,15 @@ export type Settings = {
   links?: ApolloLink[];
   resolvers?: ApolloClientOptions<null>['resolvers'];
   typeDefs?: ApolloClientOptions<null>['typeDefs'];
+  introspectionQueryResultData?: IntrospectionResultData | null;
 };
 
 class Apollo {
   settings: Required<Settings> = {
     links: [],
     resolvers: {},
     typeDefs: '',
+    introspectionQueryResultData: null,
```

Comment:
A better pattern may be to have this be `cacheConfig` (or something similar), so other params can be set besides this one.